Documentation/ceph-monitoring.md

```diff
@@ -80,6 +80,15 @@ Below the `Execute` button, ensure the `Graph` tab is selected and you should no
 You can find Prometheus Consoles here: https://github.com/ceph/cephmetrics/tree/master/dashboards/current.
 A guide to how you can write your own Prometheus consoles can be found on the official Prometheus site here: https://prometheus.io/docs/visualization/consoles/.
 
+## Prometheus Alerts
+To enable prometheus based alerting for ceph, run the following commands:  
+```
+cd cluster/examples/kubernetes/ceph/monitoring
+oc create -f prometheus-ceph-rules.yaml
+```
+
+> **NOTE** Change `Namespace: default` in `prometheus-ceph-rules.yaml` as needed. For eg. `namespace: rook-ceph` for default rook + ceph deployment.
```

Comment:
The namespace of the other example files are in `rook-ceph` Namespace so that one should be used for these new files.

Reply:
ack